setup.py

```diff
@@ -4,17 +4,17 @@
 setup(
     name='morse-talk',
     version='0.2',
-    author='Himanshu Mishra',
+    author='morse-talk developers',
     author_email='himanshu2014iit@gmail.com',
     description='An aide to Morse Code',
-    url='https://github.com/orkohunter/morse-talk',
-        download_url='https://github.com/orkohunter/morse-talk/tarball/v0.1',
-        license='MIT',
-        classifiers=[
+    url='https://github.com/morse-talk/morse-talk',
+    download_url='https://github.com/orkohunter/morse-talk/archive/master.zip',
```

Comment:
It should be

    download_url='https://github.com/morse-talk/morse-talk/archive/master.zip',